src/components/merge_results/fields/CompareProjects.tsx

```diff
@@ -0,0 +1,79 @@
+import { Divider, Typography } from '@mui/material';
+import { useContext } from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { ResearchProject } from '../../../types/project.types';
+import { Registration } from '../../../types/registration.types';
+import { MergeResultsWizardContext } from '../MergeResultsWizardContext';
+import { CompareProject } from './CompareProject';
+import { StyledEmptyProjectBox } from './ProjectBox';
+
+export const CompareProjects = () => {
+  const { t } = useTranslation();
+
+  const { control, formState } = useFormContext<Registration>();
+  const { sourceResult } = useContext(MergeResultsWizardContext);
+  const sourceProjects = sourceResult.projects;
+
+  const targetProjects = useWatch({ name: 'projects', control });
+  const initialTargetProjects = (formState.defaultValues?.projects ?? []) as ResearchProject[];
+
+  const commonProjects = sourceProjects.filter((sourceProject) =>
+    initialTargetProjects.some((targetProject) => targetProject.id === sourceProject.id)
+  );
+
+  const targetOnlyProjects = initialTargetProjects.filter(
+    (targetProject) => !commonProjects.some((sourceProject) => sourceProject.id === targetProject.id)
+  );
+
+  const addableSourceProjects = sourceProjects.filter(
+    (sourceProject) => !commonProjects.some((targetProject) => targetProject.id === sourceProject.id)
+  );
+
+  return (
+    <>
+      <Typography variant="h4" sx={{ display: { xs: 'none', md: 'block' } }}>
+        {t('registration.description.project_association')}
+      </Typography>
+      <Typography variant="h4" sx={{ gridColumn: { xs: 1, md: 3 } }}>
+        {t('registration.description.project_association')}
+      </Typography>
+
+      {initialTargetProjects.length === 0 && sourceProjects.length === 0 && (
+        <>
+          <StyledEmptyProjectBox>
+            <Typography fontStyle="italic">{t('missing_value')}</Typography>
+          </StyledEmptyProjectBox>
+          <StyledEmptyProjectBox sx={{ display: { xs: 'none', md: 'block' }, gridColumn: { xs: 1, md: 3 } }}>
+            <Typography fontStyle="italic">{t('missing_value')}</Typography>
+          </StyledEmptyProjectBox>
+          <Divider sx={{ display: { xs: 'block', md: 'none' }, my: '0.5rem' }} />
+        </>
```

Comment:
En lignende ting trengs for oversikt over filer også, så lager en egen PR som flytter dette til en egen komponent